site/_apps/react-hn-stephen.txt

```diff
@@ -0,0 +1,27 @@
+---
+weight: 9
```

Comment:
We will want to adjust the weights here to account for other apps that landed while this PR was being worked on.